specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/stable/2020-03-01/functions.json

```diff
@@ -0,0 +1,811 @@
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AzureMachineLearningWebServiceInputColumn"
+          },
+          "description": "A list of input columns for the Azure Machine Learning web service endpoint."
+        }
+      }
+    },
+    "AzureMachineLearningWebServiceInputColumn": {
+      "description": "Describes an input column for the Azure Machine Learning web service endpoint.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the input column."
+        },
+        "dataType": {
+          "type": "string",
+          "description": "The (Azure Machine Learning supported) data type of the input column. A list of valid  Azure Machine Learning data types are described at https://msdn.microsoft.com/en-us/library/azure/dn905923.aspx ."
+        },
+        "mapTo": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The zero based index of the function parameter this input maps to."
+        }
+      }
+    },
+    "AzureMachineLearningWebServiceOutputColumn": {
+      "description": "Describes an output column for the Azure Machine Learning web service endpoint.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the output column."
+        },
+        "dataType": {
+          "type": "string",
+          "description": "The (Azure Machine Learning supported) data type of the output column. A list of valid  Azure Machine Learning data types are described at https://msdn.microsoft.com/en-us/library/azure/dn905923.aspx ."
+        }
+      }
+    },
+    "JavaScriptFunctionBinding": {
+      "description": "The binding to a JavaScript function.",
+      "x-ms-discriminator-value": "Microsoft.StreamAnalytics/JavascriptUdf",
+      "allOf": [
+        {
+          "$ref": "#/definitions/FunctionBinding"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/JavaScriptFunctionBindingProperties",
+          "description": "The binding properties associated with a JavaScript function.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "JavaScriptFunctionBindingProperties": {
+      "description": "The binding properties associated with a JavaScript function.",
+      "properties": {
+        "script": {
+          "type": "string",
+          "description": "The JavaScript code containing a single function definition. For example: 'function (x, y) { return x + y; }'"
+        }
+      }
+    },
+    "FunctionRetrieveDefaultDefinitionParameters": {
+      "description": "Parameters used to specify the type of function to retrieve the default definition for.",
+      "discriminator": "bindingType",
+      "properties": {
+        "bindingType": {
+          "type": "string",
+          "description": "Indicates the function binding type."
+        }
+      },
+      "required": [
```

Comment:
>required [](http://example.com/codeflow?start=7&length=8)

Changing a property to required is a new breaking change. Was it covered by your breaking change review/approval?